### PR TITLE
Corrected the parameters of OrganisationRegistration_CompanyHouseNumb…

### PR DESCRIPTION
**Pull request by dbgoaco**
Bug fix for DP-823

### CODE REVIEW

File: Frontend/CO.CDP.OrganisationApp/Pages/Registration/CompanyHouseNumberQuestion.cshtml.cs [hunk 1]
```diff
@@ -67,7 +67,7 @@ public async Task<IActionResult> OnPost()
                 {
                     flashMessageService.SetFlashMessage(
                         FlashMessageType.Important,
-                        heading: string.Format(StaticTextResource.OrganisationRegistration_CompanyHouseNumberQuestion_CompanyAlreadyRegistered_NotificationBanner, OrganisationIdentifier, OrganisationName),
+                        heading: StaticTextResource.OrganisationRegistration_CompanyHouseNumberQuestion_CompanyAlreadyRegistered_NotificationBanner, OrganisationIdentifier, OrganisationName,
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Registration/CompanyHouseNumberQuestion.cshtml.cs:70]: Looks like some stray bits left over from the string.format

```suggestion
                        heading: StaticTextResource.OrganisationRegistration_CompanyHouseNumberQuestion_CompanyAlreadyRegistered_NotificationBanner,
```

**Review by andymantell — CHANGES_REQUESTED** [review]
`OnPost_WhenModelStateIsValidAndOrganisationAlreadyExists_ShouldShowNotificationBanner` is failing. It has a string.format which also needs removing

File: Frontend/CO.CDP.OrganisationApp/Pages/Registration/CompanyHouseNumberQuestion.cshtml.cs (hunk 1 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Registration/CompanyHouseNumberQuestion.cshtml.cs]: Fixed.

**Review by dharmverma — APPROVED** [review]

**Review by dpatel017 — APPROVED** [review]

**Review by rmohammed-goaco — APPROVED** [review]
I didn't know you could do that!

**Pull request merged**